kvdb/sqlite/driver.go

```diff
@@ -69,7 +69,8 @@ func createDBDriver(args ...interface{}) (walletdb.DB, error) {
 		return nil, err
 	}
```

Comment:
We should a check if the db already exists,  otherwise I think the followup call will just open the db.

Reply:
For the SQL backbends, we'll always call `Open`, so yeah we do need a check here to see if the db exists already.

Reply:
Pushed a commit to check if the file exists and modify what bool we pass for `isCreate`.